spec/02-integration/09-hybrid_mode/13-deprecations_spec.lua

```diff
@@ -63,6 +63,11 @@ for _, strategy in helpers.each_strategy({"postgres"}) do
         cluster_rpc = rpc,
         cluster_rpc_sync = rpc_sync,
       }))
+
+      if rpc_sync == "on" then
+        assert.logfile(dp_prefix .. "/logs/error.log").has.line("[kong.sync.v2] full sync ends", true, 20)
```

Comment:
should be `10`

Reply:
It should be. However, we already merged it into EE. This is a timeout so it will not cause any issue.